<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gjenkjenning_motta</class>
 <widget class="QWidget" name="gjenkjenning_motta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="labelOrginalFrame">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>501</width>
         <height>441</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelMatchFrame">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>30</y>
         <width>691</width>
         <height>441</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonTakePicture">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>491</y>
         <width>501</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ta bilde</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCompareImages">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>570</y>
         <width>501</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Gjenkjenn del</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>650</y>
         <width>501</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Avbryt</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMatchPercent">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>570</y>
         <width>581</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelImageName">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>650</y>
         <width>581</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonStartKamera">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>490</y>
         <width>581</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start Kamera</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
